POC-OOBE/OOBE-Sandbox/README.md

```diff
@@ -22,12 +30,30 @@ When the General Settings window is minimized, the "About Feature" section goes
 
 <img align="left" src="./images/FancyZones_extended_window_new.png" /> Extended window UI
 <img align="left" src="./images/FancyZones_smaller_window_new.png" /> Smaller window UI 
+>>>>>>> 44e3cb838e914bb1c99762e3aaa0dd01101128bb
```

Comment:
Did you forget to remove after merge?

Reply:
Thanks for  noticing that, I will clean it up